endpoints/openrtb2/auction.go

```diff
@@ -1369,16 +1370,18 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 					err = fmt.Errorf("Invalid JSON in Default Request Settings: %s", Err)
 				}
 			}
-			return nil, []error{err}
+			return nil, nil, []error{err}
 		}
 		resolvedRequest = aliasedRequest
 	}
 
 	// Apply any Stored Imps, if they exist. Since the JSON Merge Patch overrides arrays,
 	// and Prebid Server defers to the HTTP Request to resolve conflicts, it's safe to
 	// assume that the request.imp data did not change when applying the Stored BidRequest.
+	impExtInfoMap := make(map[string]exchange.ImpExtInfo)
```

Comment:
Should we set the initial capacity to `impIds`? .. or do we expect most impressions seen by PBS to not use a stored impression.

Reply:
`do we expect most impressions seen by PBS to not use a stored impression` -> yes, this is my default assumption. We now don't have many stored imps in database. 
Please refer to my comment earlier with imp array example where we want to mirror stored data in one imp out of 3.